exposures/configs/ksp-thumbsdb-disclosure-block.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-thumbsdb-disclosure-block
+  namespace: default  #use-your-ns-here
+spec:
+  message: Permission denied for thumbs.db file
+  tags : ["exposure","files"]
+  selector:
+    matchLabels:
+      app: coldfusion  #use-your-label-here
+  file:
+    matchPatterns: 
+     - pattern: /**/Thumbs.db

```

Comment:
How confident we're that the thumbs name will include capital `T` @nandakrr

Reply:
@nandakrr Kindly change that "Thumbs.db" to "thumbs.db"